diff --git a/sky130_fd_sc_hdll.tlef b/sky130_fd_sc_hdll.tlef
index 1c913ba..815b750 100644
--- a/sky130_fd_sc_hdll.tlef
+++ b/sky130_fd_sc_hdll.tlef
@@ -107,6 +107,7 @@ LAYER met1
      WIDTH 3 0.28 ;
   AREA 0.083 ;                     # Met1 6
   THICKNESS 0.35 ;
+  MINENCLOSEDAREA 0.14 ;

   ANTENNAMODEL OXIDE1 ;
   ANTENNADIFFSIDEAREARATIO PWL ( ( 0 400 ) ( 0.0125 400 ) ( 0.0225 2609 ) ( 22.5 11600 ) ) ;
@@ -149,6 +150,7 @@ LAYER met2
      WIDTH 3 0.28 ;
   AREA 0.0676 ;                       # Met2 6
   THICKNESS 0.35 ;
+  MINENCLOSEDAREA 0.14 ;

   EDGECAPACITANCE 37.759E-6 ;
   CAPACITANCE CPERSQDIST 16.9423E-6 ;

