diff -Naur sky130A/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W0p68L0p68.model.spice sky130A_patched/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W0p68L0p68.model.spice
--- sky130A/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W0p68L0p68.model.spice	2021-04-27 22:19:17.607243587 -0400
+++ sky130A_patched/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W0p68L0p68.model.spice	2021-04-27 22:21:21.018900718 -0400
@@ -21,12 +21,12 @@
 *     vary  sky130_fd_pr__pnp_05v5_W0p68L0p68__is_slope dist=gauss std=0.01662
 *   }
 * }
-.subckt  sky130_fd_pr__pnp_05v5_W0p68L0p68 c b e s
+.subckt  sky130_fd_pr__pnp_05v5_W0p68L0p68 c b e
 + 
 .param  mult = 1.0
 + sky130_fd_pr__pnp_05v5_W0p68L0p68__bf_mm = {(19.35*dkbfpp*sky130_fd_pr__pnp_05v5_W0p68L0p68__bf_slope/sqrt(mult))}
 + sky130_fd_pr__pnp_05v5_W0p68L0p68__is_mm = {(1.5075e-018*dkispp*sky130_fd_pr__pnp_05v5_W0p68L0p68__is_slope/sqrt(mult))}
-qsky130_fd_pr__pnp_05v5_W0p68L0p68 c b e s sky130_fd_pr__pnp_05v5_W0p68L0p68__model
+qsky130_fd_pr__pnp_05v5_W0p68L0p68 c b e c sky130_fd_pr__pnp_05v5_W0p68L0p68__model
 .model sky130_fd_pr__pnp_05v5_W0p68L0p68__model pnp level = 1.0
 * General Parameters
 + tref = 30.0
diff -Naur sky130A/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W3p40L3p40.model.spice sky130A_patched/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W3p40L3p40.model.spice
--- sky130A/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W3p40L3p40.model.spice	2021-04-27 22:19:17.627243532 -0400
+++ sky130A_patched/libs.ref/sky130_fd_pr/spice/sky130_fd_pr__pnp_05v5_W3p40L3p40.model.spice	2021-04-27 22:22:07.562771406 -0400
@@ -27,13 +27,13 @@
 *     vary  sky130_fd_pr__pnp_05v5_W3p40L3p40__xti_slope dist=gauss std=0.06
 *   }
 * }
-.subckt  sky130_fd_pr__pnp_05v5_W3p40L3p40 c b e s
+.subckt  sky130_fd_pr__pnp_05v5_W3p40L3p40 c b e
 + 
 .param  mult = 1.0
 + sky130_fd_pr__pnp_05v5_W3p40L3p40__bf_mm = {(16.603*dkbfpp5x*sky130_fd_pr__pnp_05v5_W3p40L3p40__bf_slope*0.45/sqrt(mult))}
 + sky130_fd_pr__pnp_05v5_W3p40L3p40__is_mm = {(7.1190e-018*1.00*dkispp5x*sky130_fd_pr__pnp_05v5_W3p40L3p40__is_slope*0.13/sqrt(mult))}
 + sky130_fd_pr__pnp_05v5_W3p40L3p40__xti_mm = {(5*sky130_fd_pr__pnp_05v5_W3p40L3p40__xti_slope/sqrt(mult))}
-qsky130_fd_pr__pnp_05v5_W3p40L3p40 c b e s sky130_fd_pr__pnp_05v5_W3p40L3p40__model
+qsky130_fd_pr__pnp_05v5_W3p40L3p40 c b e c sky130_fd_pr__pnp_05v5_W3p40L3p40__model
 .model sky130_fd_pr__pnp_05v5_W3p40L3p40__model pnp level = 1.0
 * General Parameters
 + tref = 30 subs = 1.0
